frontend/src/scenes/data-warehouse/settings/DataWarehouseManagedSourcesTable.tsx

```diff
@@ -27,6 +27,7 @@ const StatusTagSetting = {
     Completed: 'success',
     Error: 'danger',
     Failed: 'danger',
+    'Billing limits': 'danger',
```

Comment:
Maybe we should also have a link to billing page or billing documentation next to this error to explain why they reached their limit and how to expand it.

But that's something that can be added later next to the status somewhere, maybe in a tooltip?

Reply:
They will also have a banner shown at the top of the page when they hit billing limits

Reply:
neat!